portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,20 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    
+    ArrayList<String> messages = new ArrayList<String>(
```

Comment:
You could also write `ArrayList<String> messages = Listf.of("This", "Is A", "Message");`

Reply:
That's certainly a nicer way of doing it, thank you! I hadn't come across that method before. Unfortunately I don't think they introduced the List.of method until Java 9 and we've been asked to use Java 8 - but I'll definitely be using that method in my own coding from now on so thank you!

Reply:
I was not aware you were bounded to Java 8. Please ignore my comment then! Thanks!